### PR TITLE
Exclude `Rakefile.rb` from `Naming/FileName` check

### PR DESCRIPTION
**Pull request by artur-intech**
#12815

### CODE REVIEW

**Comment by bbatsov** [comment]
Don't forget to add a changelog entry as well.

**Comment by artur-intech** [comment]
> Don't forget to add a changelog entry as well.

Done! Thank you for reminding.

File: changelog/change_exclude_rakefile_rb_in_naming_filename.md [hunk 1]
```diff
@@ -0,0 +1 @@
+* [#12816](https://github.com/rubocop/rubocop/pull/12816): Ignore `Rakefile.rb` in `Naming/FileName` in the default config. ([@artur-intech][])
```
Comment (koic) [inline changelog/change_exclude_rakefile_rb_in_naming_filename.md:1]:
```suggestion
* [#12815](https://github.com/rubocop/rubocop/issues/12815): Ignore `Rakefile.rb` in `Naming/FileName` in the default config. ([@artur-intech][])
```

**Comment by koic** [comment]
Can you squash your commits into one?

**Comment by artur-intech** [comment]
Can we use Github's "Squash and merge" button?

**Comment by koic** [comment]
I apologize, but I don't wish to spend time discussing this topic. For more details, please refer to the following. Thank you.
https://github.com/rubocop/rubocop/pull/8826#issuecomment-702362621

**Pull request merged**